client/actions/goalActions.js

```diff
@@ -25,7 +25,7 @@ export function fetchGoals() { // provide all goals specific to user
 
 export function updateGoal(data) { // update a single goal
   const headers = { Authorization: `Bearer ${window.localStorage.userToken}` };
-  return dispatch => axios.put('/api/goals/update', data, { headers })
```

Comment:
this would need to be `api/goals/${SomeGoalId}`, lets remember that if we build goal-related features.